### PR TITLE
Re-adds policy_manager_impl_stress_test.cc with fix

### PR DESCRIPTION
**Pull request by JackLivio**
Adds "device" and "pre_DataConsent" to the app_policies section when the test case is building a sample policy table.

### CODE REVIEW

File: src/components/policy/test/CMakeLists.txt [hunk 1]
```diff
@@ -81,8 +81,7 @@ else ()
     sqlite_wrapper/sql_query_test.cc   
     generated_code_with_sqlite_test.cc   
 
-    # TODO{ALeshin} AssertTrue in SetUpTestCase() return false
```
Comment (JackLivio) [inline src/components/policy/test/CMakeLists.txt]: this issue has been fixed

**Comment by LuxoftAKutsan** [comment]
Reviewed

File: src/components/policy/test/policy_manager_impl_stress_test.cc [hunk 2]
```diff
@@ -143,6 +143,28 @@ void PolicyManagerImplStressTest::CreateApps(std::ofstream& ofs) {
       "]"
       "},\n";
 
+  //Device app_policies object must be present and initialized for PT to be valid.    
+  ofs << "\"device\":{\n";
+  ofs << "\"keep_context\": false,\n"
+      "\"steal_focus\": false,\n"
```
Comment (AGaliuzov) [inline src/components/policy/test/policy_manager_impl_stress_test.cc]: @JackLivio these fields  should be omitted for opensource repo

**Comment by JackLivio** [comment]
This test doesn't even exist anymore after 4.1?

**Pull request closed without merging**